Sources/Bookmarks/Migrations/BookmarkFormFactorFavoritesMigration.swift

```diff
@@ -21,15 +21,18 @@ import CoreData
 import Persistence
 import Common
 
-public class BookmarkFormFactorFavoritesMigration {
+public protocol BookmarkFormFactorFavoritesMigrating {
 
-    public enum MigrationErrors {
-        case couldNotLoadDatabase
-    }
+    func getFavoritesOrderFromPreV4Model(dbContainerLocation: URL,
+                                         dbFileURL: URL) throws -> [String]?
+}
+
+public class BookmarkFormFactorFavoritesMigration: BookmarkFormFactorFavoritesMigrating {
+
+    public init() {}
 
-    public static func getFavoritesOrderFromPreV4Model(dbContainerLocation: URL,
-                                                       dbFileURL: URL,
-                                                       errorEvents: EventMapping<MigrationErrors>? = nil) -> [String]? {
+    public func getFavoritesOrderFromPreV4Model(dbContainerLocation: URL,
+                                                dbFileURL: URL) throws -> [String]? {
```

Comment:
Instead of using event mapper to handle errors, just throw it from here and handle in client.